### ISSUE TITLE
Code search throws HTTP 500 error <Safe>: wrong type for value; expected string; got template.HTML

### ISSUE THREAD

**Issue by wolfbeast** [go-gitea/gitea#28854]
### Description

We've enabled the code indexer to be able to search source code. Trying to do so throws an error 500 with the following log entry:

```
2024/01/19 12:15:53 .../context_response.go:86:HTML() [E] Render failed: failed to render template: repo/search, error: template error: builtin(bindata):repo/search:56:87 : executing "repo/search" at <Safe>: wrong type for value; expected string; got template.HTML
												<td class="lines-code chroma"><code class="code-inner">{{.FormattedLines | Safe}}</code></td>
```

### Gitea Version

1.21.3

### Can you reproduce the bug on the Gitea demo site?

Unknown!
**note**: try.gitea.io throws a 500 so I was unable to test if the same occurs there

### Git Version

2.41.0

### Operating System

CentOS 7

### How are you running Gitea?

binary package linux amd64

### Database

MySQL/MariaDB

**Comment by lunny** [comment]
This has been fixed in 1.21.4. Please upgrade.

**Comment by wolfbeast** [comment]
> This has been fixed in 1.21.4. Please upgrade.

Ah awesome! Thank you

**Comment by wolfbeast** [comment]
Well... I will once it releases, I guess ;)
